src/brand/bhyve/boot.py

```diff
@@ -503,7 +503,8 @@ def file_or_string(f):
     bootrom = f'/usr/share/bhyve/firmware/{bootrom}'
 if not bootrom.endswith('.fd'):
     bootrom += '.fd'
-if not os.path.isfile(bootrom):
+# NOTE: boot runs in gz context, also check inside the zoneroot
+if not os.path.isfile(bootrom) and not os.path.isfile(f'{zoneroot}/{bootrom}'):
```

Comment:
To do this properly means that the brand would need to know which filesystems are automatically lookback mounted in the zone once it's booted . For example, creating `/tmp/test.fd` in the GZ would pass validation before and after this change even though the zone would not boot.

I think it's simpler just to skip validation for full paths entirely, something like:

```python
# Bootrom
bootrom = opts['bootrom']
if not ot.path.isabs(bootrom):
    bootrom = f'/usr/share/bhyve/firmware/{bootrom}'
    if not bootrom.endswith('.fd'):
        bootrom += '.fd'
    if not os.path.isfile(bootrom):
        fatal(f'bootrom {opts["bootrom"]} not found.')
```

Reply:
Seems reasonable and just tested with this code, seems to work fine with both just `BHYVE_RELEASE` or an absolute path. Updating this PR.